pkg/ocicni/ocicni.go

```diff
@@ -726,6 +759,45 @@ func (plugin *cniNetworkPlugin) GetPodNetworkStatusWithContext(ctx context.Conte
 	return results, nil
 }
 
+func (plugin *cniNetworkPlugin) GC(ctx context.Context, validPods []*PodNetwork) error {
```

Comment:
How about adding a nice doc string to this public API?

Reply:
done :-)